messages/fr-FR.json

```diff
@@ -0,0 +1,35 @@
+{
+  "store/product-quantity.quantity": "Quantité",
+  "store/specifications.title": "Caractéristiques",
+  "store/add-to-cart.success": "Article ajouté à votre panier",
+  "store/add-to-cart.duplicate": "Article déjà dans votre panier",
+  "store/add-to-cart.failure": "Un problème est survenu et l’article n’a pas été ajouté à votre panier !",
+  "store/add-to-cart.see-cart": "Voir le panier",
+  "store/add-to-cart.add-to-cart": "Ajouter au panier",
+  "store/add-to-cart.select-sku-variations": "Veuillez sélectionner une option de chaque variante",
+  "store/sku-list.inventory.title": "Stock",
+  "store/sku-list.seller.name.title": "Vendeur",
+  "store/sku-list.sku.name.title": "Nom de l’article",
+  "store/sku-list.sku.price.title": "Prix",
+  "store/sku-list.sku.quantity-selector.title": "Sélectionner la quantité",
+  "store/sku-list.seller.price.title": "Prix du vendeur",
+  "admin/editor.sku-list.highlights.title": "Points forts de l’article",
+  "admin/editor.sku-list.highlights.description": "Composant des points forts de l’article",
+  "admin/editor.sku-list.highlights.default": "Choisir le point fort par défaut",
+  "admin/editor.sku-list.highlights.allSpecifications": "Toutes les caractéristiques",
+  "admin/editor.sku-list.highlights.chooseDefaultSpecification": "Caractéristiques des champs",
+  "admin/editor.sku-list.highlights.typeSpecifications.title": "Caractéristiques de type ci-dessous",
+  "admin/editor.sku-list.highlights.showHighlight.title": "Afficher les points forts",
+  "admin/editor.sku-list.highlights.showLabel.title": "Afficher l’étiquette",
+  "admin/editor.sku-list.inventory.title": "Stock de SKU",
+  "admin/editor.sku-list.inventory.description": "Composant de stock de SKU",
+  "admin/editor.sku-list.inventory.showLabel.title": "Afficher l’étiquette",
+  "admin/editor.sku-list.item-quantity.title": "Quantité de SKU",
+  "admin/editor.sku-list.item-quantity.description": "Composant quantité de SKU",
+  "admin/editor.sku-list.item-quantity.inputType.title": "Type de sélecteur de quantité",
+  "admin/editor.sku-list.item-quantity.inputType.stepper": "Pas à pas du numérique",
+  "admin/editor.sku-list.item-quantity.inputType.number": "Saisie du numéro",
+  "admin/editor.sku-list.item-quantity.showLabel.title": "Afficher l’étiquette",
+  "admin/test": "test",
+  "store/test": "test"
```

Comment:
Are those intentional?

Reply:
We have as a source the english file so that is why we have these two strings.